ftest-toxic-reporter/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>arquillian-cube-q</artifactId>
+    <groupId>org.arquillian.cube.q</groupId>
+    <version>1.0.0.Final-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-cube-q-reporter-ftest-toxic</artifactId>
```

Comment:
Why do we need separate project for these ftests? What if I would like to add another test with slightly different configuration - should I need to create another project or somehow hack it using maven-surefire-plugin?